insights/specs/default.py

```diff
@@ -73,6 +64,7 @@ class DefaultSpecs(Specs):
     httpd_pid = simple_command("/usr/bin/pgrep -o httpd")
     openshift_router_pid = simple_command("/usr/bin/pgrep -n openshift-route")
     ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
+    rpm_format = _make_rpm_formatter()
```

Comment:
@xiangce since this is not actually a spec would it make more sense for `rpm_format`  to be moved outside of the class and renamed `_rpm_format` instead?

Reply:
@bfahr - Moving it outside of the DefaultSpecs makes sense too. Do you also think that we should move the `etc_and_sub_dirs` outside as e.g. `_etc_and_sub_dirs`?

Reply:
Oops, my bad, forgot to check the flake8 result in my local. Now it's fixed. Please check it again when free.